common/views/components/ArticleCard/ArticleCard.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import CompactCard from '../CompactCard/CompactCard';
+import Image from '../Image/Image';
+import type {Article} from '../../../model/articles';
+
+type Props = {|
+  article: Article,
+  showPosition: boolean
+|}
+
+const ArticleCard = ({ article, showPosition }: Props) => {
+  const partOfSerial = showPosition ? article.series
+    .map(series => {
+      const titles = series.schedule.map(item => item.title);
+      const positionInSerial = titles.indexOf(article.title);
```

Comment:
bit of a nitpick – could this either have the `+ 1` on the end, or be called `indexInSerial`?

Reply:
Good call. I'll leave it as index until rendering.